audit.json

```diff
@@ -496,7 +496,28 @@
-
+  "90020_Remote OS Command Injection_https://pcq.aat.platform.hmcts.net/cookies?analytics=on\u0026apm=on%26timeout+%2FT+15_GET": "ignore",
+  "10104_User Agent Fuzzer_https://pcq.aat.platform.hmcts.net/ruxitagentjs_ICA27NQVfhqrux_10271230629152232.js_GET": "ignore",
+  "10038_Obsolete Content Security Policy (CSP) Header Found_https://pcq.aat.platform.hmcts.net/accessibility-statement_GET": "ignore",
+  "10038_Obsolete Content Security Policy (CSP) Header Found_https://pcq.aat.platform.hmcts.net/cookies_GET": "ignore",
+  "10038_Obsolete Content Security Policy (CSP) Header Found_https://pcq.aat.platform.hmcts.net/cookies?lng=cy_GET": "ignore",
+  "10038_Obsolete Content Security Policy (CSP) Header Found_https://pcq.aat.platform.hmcts.net/offline_GET": "ignore",
+  "10038_Obsolete Content Security Policy (CSP) Header Found_https://pcq.aat.platform.hmcts.net/offline?lng=cy_GET": "ignore",
+  "10038_Obsolete Content Security Policy (CSP) Header Found_https://pcq.aat.platform.hmcts.net/offline?lng=en_GET": "ignore",
+  "10038_Obsolete Content Security Policy (CSP) Header Found_https://pcq.aat.platform.hmcts.net/privacy-policy_GET": "ignore",
+  "10038_Obsolete Content Security Policy (CSP) Header Found_https://pcq.aat.platform.hmcts.net/privacy-policy?lng=en_GET": "ignore",
+  "10038_Obsolete Content Security Policy (CSP) Header Found_https://pcq.aat.platform.hmcts.net/terms-conditions_GET": "ignore",
+  "10038_Obsolete Content Security Policy (CSP) Header Found_https://pcq.aat.platform.hmcts.net/terms-conditions?lng=cy_GET": "ignore",
+  "10038_Obsolete Content Security Policy (CSP) Header Found_https://pcq.aat.platform.hmcts.net/terms-conditions?lng=en_GET": "ignore",
+  "10112_Session Management Response Identified_https://pcq.aat.platform.hmcts.net_GET": "ignore",
+  "10112_Session Management Response Identified_https://pcq.aat.platform.hmcts.net/_GET": "ignore",
+  "10038_Obsolete Content Security Policy (CSP) Header Found_https://pcq.aat.platform.hmcts.net/accessibility-statement?lng=cy_GET": "ignore",
+  "10038_Obsolete Content Security Policy (CSP) Header Found_https://pcq.aat.platform.hmcts.net/cookies?analytics=on&apm=on_GET": "ignore",
+  "10038_Obsolete Content Security Policy (CSP) Header Found_https://pcq.aat.platform.hmcts.net/privacy-policy?lng=cy_GET": "ignore",
+  "10112_Session Management Response Identified_https://pcq.aat.platform.hmcts.net/offline_GET": "ignore",
+  "10112_Session Management Response Identified_https://pcq.aat.platform.hmcts.net/sitemap.xml_GET": "ignore",
+  "10038_Obsolete Content Security Policy (CSP) Header Found_https://pcq.aat.platform.hmcts.net/accessibility-statement?lng=en_GET": "ignore",
+  "10038_Obsolete Content Security Policy (CSP) Header Found_https://pcq.aat.platform.hmcts.net/cookies?lng=en_GET": "ignore",
+  "10112_Session Management Response Identified_https://pcq.aat.platform.hmcts.net/robots.txt_GET": "ignore"
```

Comment:
It feels like this tool is useless as we ignore everything - 520 or so lines of ignore.